model/trigger/version.go

```diff
@@ -106,12 +112,15 @@ func (t *versionTriggers) makeData(sub *event.Subscription) (*commonTemplateData
 	if data.PastTenseStatus == evergreen.VersionSucceeded {
 		data.PastTenseStatus = "succeeded"
 	}
+	if pastTenseOverride != "" {
```

Comment:
If `data.PastTenseStatus` is already the empty string, then this if statement is redundant.

Reply:
This is really to work around the strict formatting structure of the message, since you always need to have some verb describing what happened. EVG-3392 is open to improve this

Reply:
What I mean is that you can just `data.PastTenseStatus = pastTenseOverride` without the `if`, since if `pastTenseOverride == ""`, then it's a noop. The `if` is unnecessary.

Reply:
That causes problems for when pastTenseOverride is "" (such as the non time-based triggers) since they are setting the verb from the task status